packages/ic-asset-certification/README.md

```diff
@@ -530,7 +538,7 @@ If the configuration excludes a file that was certified, it will not be deleted.
 provided to `certify_assets` includes the Brotli and Gzip encodings, but the configuration provided to `delete_assets`
 only includes Brotli, then the Gzip file will not be deleted.
 
-Using the same base example used to demonstrate certifying assets:
+Assuming the same base example used above to demonstrate certifying assets:
```

Comment:
How about adding just a reference/link to the example above instead of copying here (and one more time below)?  The snippet for deletion is in a separated box anyway, so this is not a complete example for deletion, no?

Reply:
I'm unsure about this, because it will mean that readers have to jump up and down the docs more and it also means that when changing the original example, we have to remember that it's also being referenced in an another, unrelated section. If we have independent examples, they can change independently, if necessary.